src/main/resources/templates/hashtags.html

```diff
@@ -15,6 +15,10 @@ <h2 th:each="hashtag : ${hashtags}">
         Hashtag
     </a>
 </h2>
+<form action="/hashtags/add-reviews" method="POST">
+    <input type="text" name="title" placeholder="Hashtag name"/>
+    <button type="submit">Add Hashtag</button>
+</form>
```

Comment:
Make sure `name` attribute corresponds to the `@RequestParam` in the post mapping

Reply:
fixed via https://github.com/2020-Spring-Cohort/reviews-site-models/pull/41/commits/4a8682409c75e91bd225174719d1faef4904f7a4#diff-fd2eeff5cc477fbeadd8f39a1f01e363